apps/electron-app/src/main/menu/index.ts

```
@@ -1,59 +1,467 @@
+      { role: "resetZoom" },
+      { type: "separator" },
+      { role: "togglefullscreen" },
+      {
+        label: "Toggle Developer Tools",
+        accelerator: isMac ? "Command+Option+I" : "Control+Shift+I",
+        click: () => {
+          const focusedWindow = BrowserWindow.getFocusedWindow();
+          if (
+            focusedWindow &&
+            focusedWindow.webContents &&
+            !focusedWindow.webContents.isDestroyed()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)